# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/websocket/nsWebSocketConnection.h
# Commit: f7b6aac1b58b
# Full Hash: f7b6aac1b58bc959e431136e1b23231abe55d0b1
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-07-16 21:27:37
# Regressor Bug: 1497249
# File Overlap Count: 4
# Description:
#   Bug 1497249 - P2: ipdl for nsIWebSocketConnection r=michal
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30624
# ==============================================================================

diff -r 74bb3c2ab767 -r f7b6aac1b58b netwerk/protocol/websocket/nsWebSocketConnection.h
--- a/netwerk/protocol/websocket/nsWebSocketConnection.h	Thu Jul 16 12:59:23 2020 +0000
+++ b/netwerk/protocol/websocket/nsWebSocketConnection.h	Thu Jul 16 13:34:58 2020 +0000
@@ -33,6 +33,9 @@
                                  nsIAsyncInputStream* aInputStream,
                                  nsIAsyncOutputStream* aOutputStream);
 
+  nsresult EnqueueOutputData(nsTArray<uint8_t>&& aHeader,
+                             nsTArray<uint8_t>&& aPayload);
+
  private:
   virtual ~nsWebSocketConnection() = default;
 
